blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -81,6 +81,9 @@ protected JSONMetricsContainer createContainer(String body, String tenantId) thr
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
         try {
+
+            Tracker.track(request);
```

Comment:
@VinnyQ I just have a request - can we try adding Tracker.track calls to the base class HttpRequestHandler, so that we do not forget to add tracking for every new endpoint that we add.